### PR TITLE
Mark SDK for CI/CD as deprecated and link gpp guide

### PR DESCRIPTION
**Pull request by daniel-kurzynski**
Mark SDK for CI/CD as deprecated and link gpp guide

### CODE REVIEW

File: docs/devops/getting-started.mdx [hunk 1]
```diff
@@ -18,33 +18,8 @@ keywords:
...
-### Open SAP courses ###
-Check **Week 3** of [Create and Deliver Cloud-Native SAP S/4HANA Extensions](https://open.sap.com/courses/s4h13/items/5F9Yrn3QwX0aIFg7JmSS9H) course.
-
-### Video tutorial ###
-This video tutorial is a good example of SAP Cloud SDK for continuous delivery used with [SAP Cloud Programming Model](https://cap.cloud.sap/docs/)
-
-<div class="sdk-video-container">
-<iframe class='sdk-video' src="https://www.youtube.com/embed/Ss0mFfaE5t4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
+The SAP Cloud SDK for Continuous Delivery and its features has been migrated into [project "Piper"](../related-projects/piper.mdx).
```
Comment (stippi2) [inline docs/devops/getting-started.mdx:23]:
```suggestion
The SAP Cloud SDK for Continuous Delivery and its features have been migrated into [project "Piper"](../related-projects/piper.mdx).
```

File: docs/devops/getting-started.mdx [hunk 2]
```diff
@@ -18,33 +18,8 @@ keywords:
...
+Therefore we recommend to use the [General Purpose Pipeline of project "Piper"](https://sap.github.io/jenkins-library/stages/introduction/) instead of the SAP Cloud SDK pipeline. 
+The reasoning as well as further information how to adopt the General Purpose Pipeline are described in our [guide](https://github.com/SAP/cloud-s4-sdk-pipeline/blob/master/gpp-guide.md).
```
Comment (stippi2) [inline docs/devops/getting-started.mdx:25]:
```suggestion
The reasoning as well as further information on how to adopt the General Purpose Pipeline are described in our [guide](https://github.com/SAP/cloud-s4-sdk-pipeline/blob/master/gpp-guide.md).
```

File: sidebars.js [hunk 3]
```diff
@@ -179,7 +179,8 @@ module.exports = {
     Community: ['community/community-call'],
     'Related projects': [
       'related-projects/cloud-application-model',
-      'related-projects/sap-xsuaa-security-library-for-javascript-and-java'
+      'related-projects/sap-xsuaa-security-library-for-javascript-and-java',
+      'related-projects/project-piper'
```
Comment (stippi2) [inline sidebars.js:183]: I see the reference to the entry in the diff, but not how it then resolves.

File: docusaurus.config.js [hunk 4]
```diff
@@ -78,7 +78,7 @@ module.exports = {
                 'docs/js/release-notes-sap-cloud-sdk-for-javascript-and-typescript'
             },
             {
-              label: 'CI/CD Toolkit',
+              label: 'Continuous Delivery',
```
Comment (fwilhe) [inline docusaurus.config.js:81]: There is also a reference in line https://github.com/SAP/cloud-sdk/blob/f2cde9199590f7ae2bd8a7e480ea5c9ccefe35e8/docusaurus.config.js#L117

not sure how they relate

File: docs/devops/getting-started.mdx (hunk 1 above)
Comment (fwilhe) [inline docs/devops/getting-started.mdx:23]:
```suggestion
The SAP Cloud SDK for Continuous Delivery and its features has been merged into [project "Piper"](../related-projects/piper.mdx).
```

File: sidebars.js (hunk 3 above)
Comment (stippi2) [reply sidebars.js:183]: Did you mean 'related-projects/piper'? Because I see a "piper.mdx" file.

Comment (stippi2) [reply sidebars.js:183]: Ah, never mind. It is the ID from the .mdx file.

File: docusaurus.config.js (hunk 4 above)
Comment (daniel-kurzynski) [reply docusaurus.config.js:81]: One is for the release notes. I decided to keep it for now.
One is for the page for the Cloud SDK for Continuous Delivery. I decided to keep it and updated the content accordingly in the PR to mention the guide and the deprecation.

File: sidebars.js (hunk 3 above)
Comment (daniel-kurzynski) [reply sidebars.js:183]: yes

Comment (stippi2) [reply sidebars.js:183]: I don't have a "Resolve" button for some reason. I am logged in, as proven by the fact that this comment will be from me.

**Review by stippi2 — APPROVED** [review]

**Comment by daniel-kurzynski** [comment]
@artem-smotrakov Review and merge appreciated. Hope that reflects what we discussed. I have seen you already removed the major parts before.

**Review by fwilhe — APPROVED** [review]

**Pull request merged**